USER_GUIDE.md

```diff
@@ -1,4 +1,4 @@
-- [Getting Started with the OpenSearch Python Client](#getting-started-with-the-opensearch-python-client)
+- [Getting started with the OpenSearch python client](#getting-started-with-the-opensearch-python-client)
```

Comment:
I think Python is spelled Capital, but I'll merge as is. 

My biggest gripe with this file is those `** ***` titles ;)